packages/mesh-graphql-client/src/types.ts

```diff
@@ -0,0 +1,402 @@
+    hash: string;
+    fillableTakerAssetAmount: string;
+}
+
+export interface StringifiedAddOrdersResults {
+    accepted: StringifiedAcceptedOrderResult[];
+    rejected: StringifiedRejectedOrderResult[];
+}
+
+export interface StringifiedAcceptedOrderResult {
+    order: StringifiedOrderWithMetadata;
+    isNew: boolean;
+}
+
+export interface StringifiedRejectedOrderResult {
+    hash?: string;
+    order: StringifiedSignedOrder;
+    code: RejectedOrderCode;
+    message: string;
+}
+
+export interface StringifiedOrderEvent {
+    timestamp: string;
+    order: StringifiedOrderWithMetadata;
+    endState: OrderEventEndState;
+    fillableTakerAssetAmount: BigNumber;
+    contractEvents: ContractEvent[];
+}
+
+/**
+ * Converts StringifiedStats to Stats
+ */
+export function fromStringifiedStats(stats: StringifiedStats): Stats {
+    return {
+        ...stats,
+        latestBlock: fromStringifiedLatestBlock(stats.latestBlock),
+        maxExpirationTime: new BigNumber(stats.maxExpirationTime),
+        startOfCurrentUTCDay: new Date(stats.startOfCurrentUTCDay),
+    };
+}
+
+/**
+ * Converts StringifiedLatestBlock to LatestBlock
+ */
+export function fromStringifiedLatestBlock(latestBlock: StringifiedLatestBlock): LatestBlock {
+    return {
+        ...latestBlock,
+        number: new BigNumber(latestBlock.number),
+    };
+}
+
+/**
+ * Converts SignedOrder to StringifiedSignedOrder
+ */
+export function toStringifiedSignedOrder(order: SignedOrder): StringifiedSignedOrder {
+    return {
+        ...order,
+        chainId: order.chainId.toString(),
+        makerAssetAmount: order.makerAssetAmount.toString(),
+        takerAssetAmount: order.takerAssetAmount.toString(),
+        makerFee: order.makerFee.toString(),
+        takerFee: order.takerFee.toString(),
+        expirationTimeSeconds: order.expirationTimeSeconds.toString(),
+        salt: order.salt.toString(),
+    };
+}
+
+/**
+ * Converts StringifiedOrderWithMetadata to OrderWithMetadata
+ */
+export function fromStringifiedOrderWithMetadata(order: StringifiedOrderWithMetadata): OrderWithMetadata {
+    return {
+        ...order,
+        // tslint:disable-next-line: custom-no-magic-numbers
+        chainId: Number.parseInt(order.chainId, 10),
+        makerAssetAmount: new BigNumber(order.makerAssetAmount),
+        takerAssetAmount: new BigNumber(order.takerAssetAmount),
+        makerFee: new BigNumber(order.makerFee),
+        takerFee: new BigNumber(order.takerFee),
+        expirationTimeSeconds: new BigNumber(order.expirationTimeSeconds),
+        salt: new BigNumber(order.salt),
+        fillableTakerAssetAmount: new BigNumber(order.fillableTakerAssetAmount),
+    };
+}
+
+/**
+ * Converts StringifiedSignedOrder to SignedOrder
+ */
+export function fromStringifiedSignedOrder(order: StringifiedSignedOrder): SignedOrder {
+    return {
+        ...order,
+        // tslint:disable-next-line: custom-no-magic-numbers
```

Comment:
I'd be okay with getting rid of this linter rule entirely.

Reply:
We can't do that easily because of how we pull in the linter config from `@0x/tslint-config`. I say we just keep it. This also helps keep our code here _sort of_ in the same style as the monorepo.

Reply:
Alternatively if you want to remove this rule from `@0x/tslint-config` you should ask what other people think.

Reply:
It's actually relatively simple. We have disabled this within the `contracts/` directory of the monorepo using this `tslint.json`: https://github.com/0xProject/0x-monorepo/blob/development/contracts/exchange/tslint.json. 

I'm okay with not making this change in this PR, but I think we should disable `custom-no-magic-numbers` eventually since it doesn't correlate with acceptable use of magic numbers.